randommaster.py

```diff
@@ -0,0 +1,95 @@
+Python 3.11.0 (main, Oct 24 2022, 18:26:48) [MSC v.1933 64 bit (AMD64)] on win32

```

Comment:
Why is this being submitted as a new file?

Reply:
because I don't know how to use Github!! hahaha

Reply:
Basically just modify the existing file, commit your changes and then push to your own repo.  Then submit PR.  

As to the code, I would suggest you pass the array into the search function and then randomize the selection at the point where the actual reply is conducted.  That way it will randomly select each reply.